ui/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.container.js

```diff
@@ -14,19 +15,25 @@ function convertGasLimitForInputs(gasLimitInHexWEI) {
   return parseInt(gasLimitInHexWEI, 16) || 0;
 }
 
+function convertMinimumGasLimitForInputs(minimumGasLimit) {
+  return parseInt(minimumGasLimit, 10) || Number(MIN_GAS_LIMIT_DEC);
```

Comment:
Very minor non-blocking nit: if `minimumGasLimit` is undefined, then `parseInt(minimumGasLimit, 10)` will return `NaN`. This is okay because it is falsy and so this function will return `Number(MIN_GAS_LIMIT_DEC)`

This function might be conceptually simpler if we default an undefined `minimumGasLimit` param to `MIN_GAS_LIMIT_DEC` and remove the need to default to a call to `Number`

This way no piece of this ever produces `NaN`. Also, if for some reason we ever do want to pass `0` as the `minimumGasLimit`, then that would be supported as well (current logic would result in `MIN_GAS_LIMIT_DEC` being return if the function is passed `0`)

Reply:
Good suggestion! fixed